examples/haproxy/haproxy.cfg

```diff
@@ -59,6 +59,8 @@ backend berghain_http
     acl has_token var(txn.berghain.token) -m found
 
     http-after-response add-header set-cookie "berghain=%[var(txn.berghain.token)]; path=/;" if has_token
+    # or, if using the "trusted_domains" option in Berghain (conditionalize in case of multiple entries)
+    #http-after-response add-header set-cookie "berghain=%[var(txn.berghain.token)]; domain=example.com; path=/;" if has_token
```

Comment:
We could always return the domain and template it into the haproxy config. That way we don't need a special case in haproxy and the overhead is minimal

Reply:
You mean setting it in a berghain.foo variable through the SPOE?